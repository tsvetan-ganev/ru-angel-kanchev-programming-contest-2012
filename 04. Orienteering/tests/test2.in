25 24
2303Mutafova w 08:37:24 09:58:11
0224Kirilov m 08:18:25 09:20:19
1016Bozhkov m 08:18:48 09:44:38
0323Miteva w 08:35:49 10:03:51
1115Nikolova w 10:32:59 11:42:49
0719Petrova w 10:13:04 11:47:52
0110Mihaylov m 10:22:24 11:23:55
1313Enchev m 09:59:28 11:06:30
2204Mogilska w 08:18:25 10:01:14
1610Gotseva w 10:15:41 11:18:36
2501Keremedchieva w 09:51:52 11:35:07
0917Vlaskovski m 10:50:12 12:31:37
1907Iliyanova w 08:39:48 09:52:32
2402Kateliev m 09:51:51 11:44:09
0620Ivanova w 08:11:10 09:03:59
1808Marinov m 08:06:44 10:17:59
1214Dimitrov m 10:08:01 11:28:09
0818Stefanova w 08:42:38 10:36:24
1709Shakir m 10:54:52 12:55:24
1511Stefanov m 09:09:11 10:11:30
2105Petrov m 08:37:54 09:56:41
2006Atanasova w 10:49:17 12:21:41
1412Vedat m 09:30:50 11:20:20
0422Ploshtakov m 09:31:48 10:41:42
0521Pavlov m 08:07:30 09:32:22