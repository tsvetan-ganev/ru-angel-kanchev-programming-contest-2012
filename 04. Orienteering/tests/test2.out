01 02 15 
06 16 11 
10 